src/index.js

```diff
@@ -74,7 +74,7 @@ _adapters._date.override({
     if (unit === 'isoWeek') {
       weekday = Math.trunc(Math.min(Math.max(0, weekday), 6));
       const dateTime = this._create(time);
-      return dateTime.minus({days: (dateTime.weekday < weekday ? 7 : 0) + dateTime.weekday - weekday}).startOf('day').valueOf();
+      return dateTime.minus({days: (dateTime.weekday % 7) - weekday}).startOf('day').valueOf();
```

Comment:
@Billiam I think to solve the issue of Sunday week day (7 for Luxon), the right fix could be:

`return dateTime.minus({days: (dateTime.weekday < weekday ? 7 : 0) + (dateTime.weekday % 7) - weekday}).startOf('day').valueOf();`

Reply:
I'd use:
```js
return dateTime.minus({days: (7 + dateTime.weekday - weekday) % 7}).startOf('day').valueOf();
```

Reply:
To verify, you can just log all the combinations:
```js
for (let lx = 1; lx < 8 ; lx++) { 
  for (let iso = 0; iso < 7 ; iso++) { 
    console.log({from_luxon: lx, to_iso: iso, minus: (7 + lx - iso) % 7}); 
  }
}
```

Reply:
Nice :+1: